conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -181,24 +135,23 @@ public void endSession() {
     public void updateContentMetadata(MetadataOverrides metadataOverrides) {
         this.contentMetadataBuilder.setOverrides(metadataOverrides);
         this.metadataOverrides = metadataOverrides;
-
-        if (!this.isSessionActive()) {
-            Log.i(TAG, "[ ConvivaAnalytics ] no active session; Don't propagate content metadata to conviva.");
-            return;
-        }
-
         this.createContentMetadata();
         this.updateSession();
     }
 
+    public void release() {
+        convivaVideoAnalytics.release();
+        ConvivaAnalytics.release();
```

Comment:
Is there a need to release convivaAdAnalytics object as well?

Reply:
Ah yes, we're not actually using AdAnalytics at the moment so that should probably be removed for now.